lib/topics_filter.rb

```diff
@@ -18,7 +18,7 @@ def filter_from_query_string(query_string)
     filters = {}
 
     query_string.scan(
-      /(?<key_prefix>[-=])?(?<key>[\w-]+):(?<value>[^\s]+)/,
+      /(?<key_prefix>(?:-|=|-=|=-))?(?<key>[\w-]+):(?<value>[^\s]+)/,
```

Comment:
Is it right? My understanding is:
`-` - exclude category and subcategories
`=` - include exact category
`-=` - exclude exact category
`=-` - ?

Reply:
`-=` and `=-` does the sam thing :)